### ISSUE TITLE
NTR: Radiculomegaly

### ISSUE THREAD

**Issue by callahantiff** [obophenotype/human-phenotype-ontology#6221]
**Preferred term label:**
Radiculomegaly

**Synonyms**
Root gigantism (`exact`), Long root (`exact`), Tooth gigantism (`broad`)

**Definition (free text, please give PubMed ID)**
Elongated tooth root. [PMID:29574060](https://pubmed.ncbi.nlm.nih.gov/29574060/)

**Parent term (use hpo.jax.org/app)**
Abnormal size of the dental root (`HP_0040220`)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
- Oculofaciocardiodental syndrome ([`ORPHA:2712`](https://www.orpha.net/consor/cgi-bin/OC_Exp.php?Lng=EN&Expert=2712), [`OMIM_300166`](http://omim.org/entry/300166))  
- Marashi-Gorlin syndrome [`ORPHA:3013`](https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert=3013)  

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-8169-9049

**Comment by pnrobinson** [comment]
added as HP_0033189. Will add annotations after next release
